Makefile

```diff
@@ -149,13 +140,19 @@ push-docker-images: docker-image bootnode-image
 	docker push $(BOOTNODE_IMAGE_NAME):$(DOCKER_IMAGE_CUSTOM_TAG)
 	docker push $(DOCKER_IMAGE_NAME):$(DOCKER_IMAGE_CUSTOM_TAG)
 
-push-docker-images-latest: docker-image bootnode-image
+# See https://www.gnu.org/software/make/manual/html_node/Target_002dspecific.html to understand this magic.
+push-docker-images-latest: GIT_BRANCH = $(shell git rev-parse --abbrev-ref HEAD)
+push-docker-images-latest: GIT_LOCAL  = $(shell git rev-parse @)
+push-docker-images-latest: GIT_REMOTE = $(shell git fetch -q && git rev-parse remotes/origin/develop || echo 'NO_DEVELOP')
+push-docker-images-latest:
```

Comment:
`docker-image bootnode-image` dependencies were removed. Was that intentional?

Reply:
ouch! no, it was accidental. thanks!